### ISSUE TITLE
Consider support for sections like Pronunciation at top-level for multi-word entries

### ISSUE THREAD

**Issue by lgessler** [lgessler/wdp#2]
When an entry for a word has multiple etymologies, some sections like `Pronunciation` can be included either at level 3 as `===Pronunciation===`, presumably applying to all etymologies, or at level 4 `====Pronunciation====` under an etymology. Compare [the English entry for _set_](https://en.wiktionary.org/wiki/set#English), where `===Pronunciation===` is at level 3, with [the Catalan entry for _set_](https://en.wiktionary.org/wiki/set#Catalan), where `====Pronunciation====` is at level 4.

As of 1c9844f wdp always performs the latter. We will need to consider an extension if people want sections like Pronunciation at level 3 with multi-word entries.

**Comment by aryamanarora** [comment]
Makes sense for languages that use abjads (e.g. I think this is common practice for Arabic on Wiktionary).
